factorial.java

```diff
@@ -0,0 +1,16 @@
+import java.util.*;
+public class factorial{
```

Comment:
Please Rename the class name and file name as well to `Factorial`. The initial letter should be in caps for the Java class.

Reply:
Updated

Reply:
Thank you 👍